drivers/hmis/app/models/hmis/form/definition_item_filter.rb

```diff
@@ -59,6 +62,9 @@ def eval_rule(rule)
     # but should always be shown when creating/editing a Client outside of a project context.
     return true if project.nil?
 
+    # TODO we should probably add an "always show" override. there are workarounds e.g. selecting all project types
+    # return true if rule.fetch('all_projects', false) == true
```

Comment:
Thoughts on this? Didn't do it directly because it was complicated to add to the JSON Schema

Reply:
One way to to do this would be to add a 'MATCH' operator? Then we'd get this as a side-effect with:
```ruby
{ variable: 'projectType', operator: 'MATCH', value: ".*" }
```

I am a little surprised that it's a common need to ignore the hud rules

Reply:
Actually we probably don't need this. We only do it for the QA forms where we want to show everything all the time.  I'll remove this comment